tests/integration/test_orchestration.py

```diff
@@ -4,7 +4,6 @@
 on another container, and so on, then all the contains should be built in the
 right order.
 """
-
```

Comment:
Given there are no changes to this file any more can we rever the whitespace change as well 😄

Reply:
Of course